src/components/VariantItem/VariantItem.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { withStyles } from "material-ui/styles";
+import Button from "material-ui/Button";
+import Typography from "material-ui/Typography";
+import classNames from "classnames";
+
+const styles = () => ({
+  variantButton: {
+    "display": "flex",
+    "justifyContent": "space-between",
+    "textTransform": "none",
+    "width": "100%",
+    "transition": "opacity 200ms linear",
+    "borderRadius": "2px",
+    "backgroundColor": "#fff",
+    "border": "1px solid #fff",
+    "padding": "10px 20px",
```

Comment:
use some multiple of `theme.spacing.unit` so if we change the global spacing unit, all components will change as well.